xml/deploy_cephadm.xml

```diff
@@ -702,6 +702,25 @@ o- time_server ................................................ [enabled]
      path, add it as follows:
     </para>
 <screen>&prompt.smaster;ceph-salt config /cephadm_bootstrap/ceph_image_path set registry.suse.com/ses/7/ceph/ceph</screen>
+   <note>
+     <para>
+       For the monitoring stack, there are more container images needed. For an
+       air-gapped deployment as well as deploying from a local registry, you may
+       want to get those images at this point to prepare your local registry
+       accordingly.
+     </para>
+     <para>
+       Please note that they will not be used by &cephsalt; for the deployment.
+       Deploying monitoring services is part of the Operations and
```

Comment:
- really the Admin guide? i thought it's https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-day2-service-monitoring
- anyways, attach corresponding link please